src/containers/Devices.jsx

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux'
```

Comment:
To be honest, I don't think it is. As you say, I think it would be easier to maintain with the view and the container as one entity.
However, this is how the existing Accounts view is structured, and I think both should be consistent. I'm quite in favor of merging `Devices` and `DevicesView`, but then the same needs to be done for `Account` and `AccountView`. I don't mind doing them both.